src/routes/api/trips.routes.js

```diff
@@ -0,0 +1,137 @@
+import express from 'express';
+import tripsController from '../../controllers/tripsController';
+import checkUser from '../../middlewares/checkUser';
+import tripRequestValidator from '../../middlewares/tripRequestValidator';
+
+const router = express.Router();
+
+/**
+ * @swagger
+ *
+ * /api/v1/trip/request:
+ *   post:
+ *     security: []
+ *     summary: one way trip requests
+ *     description: one way trip requests
+ *     tags:
+ *       - Users
+ *     requestBody:
+ *       content:
+ *         application/json:
+ *           schema:
+ *             type: object
+ *             properties:
+ *               type:
+ *                 type: string
+ *               passportNumber:
+ *                 type: string
+ *               gender:
+ *                 type: string
+ *               fromCountry:
+ *                 type: string
+ *               fromCity:
+ *                 type: string
+ *               toCountry:
+ *                 type: string
+ *               toCity:
+ *                 type: string
+ *               accommodation:
+ *                 type: string
+ *               departureDate:
+ *                 type: string
+ *               returnDate:
+ *                 type: string
+ *               reason:
+ *                 type: string
+ *     produces:
+ *         application/json:
+ *           schema:
+ *             type: object
+ *             properties:
+ *               status:
+ *                 type: integer
+ *               message:
+ *                 type: string
+ *               data:
+ *                 type: object
+ *                 properties:
+ *                   status:
+ *                     type: integer
+ *                   passportNumber:
+ *                     type: string
+ *                   from:
+ *                     type: string
+ *                   to:
+ *                     type: string
+ *                   reason:
+ *                     type: string
+ *     responses:
+ *       201:
+ *         description: Travel request successfully created
+ */
+
+/**
+ * @swagger
+ *
+ * /api/v1/trip/request:
+ *   post:
+ *     security: []
+ *     summary: return trip requests
+ *     description: return trip requests
+ *     tags:
+ *       - Users
+ *     requestBody:
+ *       content:
+ *         application/json:
+ *           schema:
+ *             type: object
+ *             properties:
+ *               type:
+ *                 type: string
+ *               passportNumber:
```

Comment:
```suggestion
 *               passportNumber:
                          type: string
                          example: RK0884756
```
Would you please try to add an example for every key and value property? because it would be easier for us to have a clue on what to enter while we are testing this in ```swagger```

Reply:
@hakoemmy  great

Reply:
@blaise82 Thank you for the response.

Reply:
@hakoemmy  done

Reply:
@hakoemmy  thanks for the feed back too